src/main/java/org/folio/service/finance/transaction/EncumbranceService.java

```diff
@@ -160,7 +169,8 @@ public Future<List<Transaction>> getOrderEncumbrancesToUnrelease(CompositePurcha
     // Check invoice line's releaseEncumbrance value for each order line
     List<Future<List<Transaction>>> futures =
       compPO.getCompositePoLines()
-        .stream().filter(poLines->poLines.getFundDistribution().size() > 0)
+          .stream()
+          .filter(poLines -> !poLines.getFundDistribution().isEmpty())
```

Comment:
Can be replaced to CollectionUtils.isNotEmpty(poLines.getFundDistribution())

Reply:
agree